setup.py

```diff
@@ -70,6 +70,7 @@
     python_requires=">=3.7",
     extras_require={
         'pyscf': ["pyscf; python_version < '3.10' and sys_platform != 'win32'"],
+        'visualization':["matplotlib>=3.3"],
```

Comment:
```suggestion
        'mpl':["matplotlib>=3.3"],
```
I believe this is also what Terra calls it, so I suggest to keep things aligned as much as possible